typed-builder-macro/src/struct_info.rs

```diff
@@ -285,13 +287,76 @@ impl<'a> StructInfo<'a> {
                 #[deprecated(
                     note = #repeated_fields_error_message
                 )]
-                pub fn #field_name (self, _: #repeated_fields_error_type_name) -> #builder_name <#( #target_generics ),*> {
+                pub fn #method_name (self, _: #repeated_fields_error_type_name) -> #builder_name <#( #target_generics ),*> {
                     self
                 }
             }
         })
     }
 
+    fn method_name(field: &FieldInfo, field_name: &Ident) -> Result<Ident, Error> {
+        let prefix = match &field.builder_attr.setter.prefix {
+            Some(prefix) => match prefix {
+                syn::Expr::Lit(lit) => match &lit.lit {
+                    syn::Lit::Str(str) => Some(syn::Ident::new(&format!("{}", str.value()), proc_macro2::Span::call_site())),
+                    _ => {
+                        return Err(Error::new_spanned(
```

Comment:
Wouldn't it make more sense to do this check in `SetterSettings::apply_meta`?

Reply:
Also, this whole thing (converting an `syn::Expr` to a string) should probably be extracted to a function.